daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/StablePackages.scala

```diff
@@ -0,0 +1,120 @@
+    ) -> PackageId.assertFromString(
+      "518032f41fd0175461b35ae0c9691e08b4aea55e62915f8360af2cc7a1f2ba6c"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Date-Types-bfcd37bd6b84768e86e432f5f6c33e25d9e7724a9d42e33875ff74f6348e733f"
+    ) -> PackageId.assertFromString(
+      "bfcd37bd6b84768e86e432f5f6c33e25d9e7724a9d42e33875ff74f6348e733f"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Internal-Any-cc348d369011362a5190fe96dd1f0dfbc697fdfd10e382b9e9666f0da05961b7"
+    ) -> PackageId.assertFromString(
+      "cc348d369011362a5190fe96dd1f0dfbc697fdfd10e382b9e9666f0da05961b7"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Internal-Down-057eed1fd48c238491b8ea06b9b5bf85a5d4c9275dd3f6183e0e6b01730cc2ba"
+    ) -> PackageId.assertFromString(
+      "057eed1fd48c238491b8ea06b9b5bf85a5d4c9275dd3f6183e0e6b01730cc2ba"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Internal-Template-d14e08374fc7197d6a0de468c968ae8ba3aadbf9315476fd39071831f5923662"
+    ) -> PackageId.assertFromString(
+      "d14e08374fc7197d6a0de468c968ae8ba3aadbf9315476fd39071831f5923662"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Logic-Types-c1f1f00558799eec139fb4f4c76f95fb52fa1837a5dd29600baa1c8ed1bdccfd"
+    ) -> PackageId.assertFromString(
+      "c1f1f00558799eec139fb4f4c76f95fb52fa1837a5dd29600baa1c8ed1bdccfd"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Monoid-Types-6c2c0667393c5f92f1885163068cd31800d2264eb088eb6fc740e11241b2bf06"
+    ) -> PackageId.assertFromString(
+      "6c2c0667393c5f92f1885163068cd31800d2264eb088eb6fc740e11241b2bf06"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-NonEmpty-Types-e22bce619ae24ca3b8e6519281cb5a33b64b3190cc763248b4c3f9ad5087a92c"
+    ) -> PackageId.assertFromString(
+      "e22bce619ae24ca3b8e6519281cb5a33b64b3190cc763248b4c3f9ad5087a92c"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Semigroup-Types-8a7806365bbd98d88b4c13832ebfa305f6abaeaf32cfa2b7dd25c4fa489b79fb"
+    ) -> PackageId.assertFromString(
+      "8a7806365bbd98d88b4c13832ebfa305f6abaeaf32cfa2b7dd25c4fa489b79fb"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Set-Types-97b883cd8a2b7f49f90d5d39c981cf6e110cf1f1c64427a28a6d58ec88c43657"
+    ) -> PackageId.assertFromString(
+      "97b883cd8a2b7f49f90d5d39c981cf6e110cf1f1c64427a28a6d58ec88c43657"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Time-Types-733e38d36a2759688a4b2c4cec69d48e7b55ecc8dedc8067b815926c917a182a"
+    ) -> PackageId.assertFromString(
+      "733e38d36a2759688a4b2c4cec69d48e7b55ecc8dedc8067b815926c917a182a"
+    ),
+    PackageName.assertFromString(
+      "daml-stdlib-DA-Validation-Types-99a2705ed38c1c26cbb8fe7acf36bbf626668e167a33335de932599219e0a235"
+    ) -> PackageId.assertFromString(
+      "99a2705ed38c1c26cbb8fe7acf36bbf626668e167a33335de932599219e0a235"
+    ),
+  )
+  val ids = map.values.toSet
```

Comment:
```suggestion
  val Ids = map.values.toSet
```
(in the spirit of the [Scala style guide](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables))